azure-iot-device/azure/iot/device/iothub/sync_clients.py

```diff
@@ -315,6 +332,8 @@ def send_message(self, message):
         If the connection to the service has not previously been opened by a call to connect, this
         function will open the connection before sending the event.
 
+        This method is not compatible with PNP.
```

Comment:
>PNP [](http://example.com/codeflow?start=43&length=3)

This wording may have to change.  We're not using "PNP" in any API surfaces.  I'm not sure about documentation.